liferay-echo/liferay-portal
# LPS-168592 Implements Import Action for Utility Pages

## Motivation

[Link to task](https://issues.liferay.com/browse/LPS-168592)

For utility pages we also want the users to export/import them just like with page templates, master pages and display pages. This PR handles this.


## Change summary:

* Create a new action command so that the user can export utility pages
* Modify the existing import class so that we can also import utility pages
* Remove validators from api module and move them to impl module since they shouldn't be exposed


## Test Scenario

* Enable feature flag: feature.flag.LPS-162765=true
* Go to Site Builder Pages > Utility Pages
* Create a new 404 utility page and add some fragments
* Click on the kebab menu and export it
* Delete the utility page and re-import it
* The imported utility page should have all the fragments

![import](https://user-images.githubusercontent.com/4267448/203339021-5e0f3de7-0b72-4a07-b477-9f1a10eb06f8.gif)
